src/style.css

```diff
@@ -1,11 +1,12 @@
-@import "tailwindcss";
+@import "tailwindcss" source(".");
```

Comment:
This change is what allowed me to fix the issue with the primary color not being able to be set.